packages/block-library/src/navigation-link/style.scss

```diff
@@ -93,12 +97,11 @@
 		// NOTE TO DEVS - if refactoring this code, please double-check that
 		// submenus have a default background color, this feature has regressed
 		// several times, so care needs to be taken.
-		background-color: $white;
-		color: $gray-900;
+		background-color: #fff;
```

Comment:
Since this is the style.scss file, it should use plain hex colors, not the editor UI variables.

Reply:
We definitely should. In the mean time, this was a teensy "while I'm here" change.